packages/editor/src/components/provider/index.js

```diff
@@ -28,6 +28,12 @@ const { ExperimentalBlockEditorProvider } = unlock( blockEditorPrivateApis );
 const { PatternsMenuItems } = unlock( editPatternsPrivateApis );
 
 const noop = () => {};
+const NON_CONTEXTUAL_POST_TYPES = [
```

Comment:
Would be helpful to explain further what these are or why they are distinct

Reply:
Follow-up here https://github.com/WordPress/gutenberg/pull/58836